automation/src/test/scala/org/broadinstitute/dsde/test/api/BillingProjectOwnerPermissionSpec.scala

```diff
@@ -0,0 +1,270 @@
+package org.broadinstitute.dsde.test.api
+
+import org.broadinstitute.dsde.test.util.AuthDomainMatcher
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.config.{Credentials, UserPool}
+import org.broadinstitute.dsde.workbench.fixture.{BillingFixtures, GroupFixtures, WorkspaceFixtures}
+import org.broadinstitute.dsde.workbench.service.BillingProject.BillingProjectRole
+import org.broadinstitute.dsde.workbench.service.Orchestration.groups.GroupRole
+import org.broadinstitute.dsde.workbench.service.{Rawls, RestException, Sam}
+import org.scalatest.{FreeSpec, Matchers}
+
+import scala.util.Try
+
+class BillingProjectOwnerPermissionSpec extends FreeSpec with WorkspaceFixtures with BillingFixtures with GroupFixtures  with Matchers {
+
+  /*
+ * Unless otherwise declared, this auth token will be used for API calls.
+ * We are using a curator to prevent collisions with users in tests (who are Students and AuthDomainUsers), not
+ *  because we specifically need a curator.
+ */
+
+  val defaultUser: Credentials = UserPool.chooseCurator
+  val authTokenDefault: AuthToken = defaultUser.makeAuthToken()
```

Comment:
All of the tests in this class declare their own implicit auth token so this isn't actually being used anywhere

Reply:
@ansingh7115 Intentionally, I wrote new tests as close to as original `AuthDomainOwnersSpec` so I don't accidentally changed original tests' logic. Just want to be sure, are we sure about making suggested change? I do think suggested changes are going to help to make tests clearer and cleaner.

Reply:
removed `defaultUser` and `authTokenDefault`